### PR TITLE
[Delivers #106953230] no inactive users in approvers

### PR DESCRIPTION
**Pull request by pkarman**
Story: https://www.pivotaltracker.com/story/show/106953230

### CODE REVIEW

**Comment by pkarman** [comment]
depends on #792 -- merge that first.

**Comment by jessieay** [comment]
793 was merged - do you mind pulling in changes from master so that the changes specific to this PR are more obvious?

**Comment by pkarman** [comment]
@jessieay done

File: spec/features/ncr/work_orders/create_spec.rb [hunk 1]
```diff
@@ -112,6 +112,17 @@
       expect(page).to_not have_content("Approving official email can't be blank")
     end
 
+    scenario "inactive users do not appear as potential approvers", :js do
```
Comment (jessieay) [inline spec/features/ncr/work_orders/create_spec.rb]: there is a test for this here: https://github.com/18F/C2/pull/734/files#diff-9965b424c2f291874f17f5d65d09fc9dR1

if that was not a good / obvious place for it, should we perhaps remove it alongside adding this?

Comment (pkarman) [inline spec/features/ncr/work_orders/create_spec.rb]: I don't think it hurts to have more tests?

The one I added does use the `:js` feature and clickety stuff to make sure DOM is not hiding it. Not sure if that matters.

Comment (jessieay) [inline spec/features/ncr/work_orders/create_spec.rb]: Doesn't hurt to have more, but these tests are pretty much duplicates in different locations. I think it makes sense to have one with js!

**Comment by pkarman** [comment]
If you think they're duplicates, let's just close this PR and the story and call it a day. I obviously didn't look closely enough before adding the test.

**Comment by pkarman** [comment]
Ok, I have removed the inactive users spec file, since it only had one test, now duplicated with `:js` support in NCR-specific file.

**Comment by jessieay** [comment]
looks great! merging
